readme/BREAKING_CHANGES.markdown

```diff
@@ -841,4 +841,26 @@ implementation and use it locally within your module.
 This change was made to enable image drag and drop handling in CKEditor, and
 have a common solution for both Alloy Editor and CKEditor.
 
+---------------------------------------
+
+### Moving lexicon icons path
```

Comment:
@jbalsas is this what you were thinking about adding? This was the only BREAKING file I saw

Reply:
sounds good enough to me, yeah! 👍